engine/class_modules/sc_hunter.cpp

```diff
@@ -6242,6 +6246,9 @@ struct melee_focus_spender_t: hunter_melee_attack_t
     double am = hunter_melee_attack_t::action_multiplier();
 
     am *= 1 + p() -> buffs.mongoose_fury -> stack_value();
+        
+    if ( p()->buffs.strike_it_rich->check() )
+      am *= 1 + p() -> buffs.strike_it_rich -> value().percent();  
```

Comment:
value() is a double so this won't compile, percent() is a function of a spelldata effect to pull the effect value interpreted as a percent (usually value / 100)
since you're pulling the damage bonus from the buff which has the value set from the spelldata effect already, you won't need to do anything else
the alternative where percent() would be needed is instead of looking at the buff you look at the spelldata effect, examples are elsewhere in the code but basically would look like p()->tier_set.tww_s2_sv_4pc_buff.effectN( 1 ).percent()